plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -235,8 +235,9 @@
     private final MapType varcharMapType;
     private final List<String> tableTypes;
     private final AggregateFunctionRewriter<JdbcExpression> aggregateFunctionRewriter;
+    private final boolean stringPushdownWithCollate;
```

Comment:
Maybe `isStringPushdownWithCollateEnabled`?

Reply:
Revised to `enableStringPushdownWithCollate`